.gitlab-ci.yml

```diff
@@ -33,6 +33,7 @@ gpu:test:dev:
   extends: .gputest
   variables:
     CI_VERSION_TAG: 'dev'
+  allow_failure: true
```

Comment:
Just an FYI that this means we are effectively turning off GPU CI for the balance law solver.

Reply:
@simonbyrne why was this necessary?

Reply:
See #232